praw/praw.ini

```diff
@@ -13,7 +13,7 @@ trophy_kind=t6
 # The URL prefix for OAuth-related requests.
 oauth_url=https://oauth.reddit.com
 
-# The amount of seconds to ratelimit
+# The amount of seconds of ratelimit to sleep for silently
```

Comment:
This still isn't a very clear comment. I think I know what this rate limit is used for, but it's really not clear from this comment when it's applied. Maybe something like

`The maximum number of seconds that PRAW will automatically sleep for when Reddit returns a specific "429 you are doing that too much" response.

Reply:
It doesn't catch 429s, but an APIException.